include/mbedtls/build_info.h

```diff
@@ -161,6 +161,35 @@
 #define MBEDTLS_PK_PARSE_EC_COMPRESSED
 #endif
 
+/* PK module can achieve ECDH and ECDSA functionalities by means of either
```

Comment:
Thanks for the comment, but this is no entirely correct: PK has no (EC)DH functionality. We've use the PK namespace for `MBEDTLS_PK_CAN_ECDH` because there was not better place, and ECDH is still asymmetric (aka Public Key) crypto, so conceptually close to PK... Also, PK is the lowest part of the stack that obeys `USE_PSA`. However perhaps that was a bad idea and we should go with `MBEDTLS_CAN_ECDH`?

Anyway, I suggest keeping this comment, stripped of references to ECDH, on top of the block about ECDSA.

Then having another comment above the ECDH block. If we keep PK in the name, the comment should explain how that's a stretch, but then again perhaps we shouldn't pick names that require explanatory comments if we can avoid it :)

Reply:
> However perhaps that was a bad idea and we should go with MBEDTLS_CAN_ECDH?

Right, now that you mentioned this, I checked and this symbol is not used at all in PK module so it doesn't make much sense to keep `PK` in the symbol's name.